research/picai/preprocessing_transforms.py

```diff
@@ -0,0 +1,176 @@
+from typing import Optional, Sequence, Tuple, Union
+
+import numpy as np
+import SimpleITK as sitk
+
+
+def resample_img(
+    image: sitk.Image,
+    spacing: Sequence[float] = (2.0, 2.0, 2.0),
+    size: Optional[Sequence[int]] = None,
+    is_label: bool = False,
+    pad_value: Optional[Union[float, int]] = 0.0,
+) -> sitk.Image:
+    """
+    Resample images to target resolution spacing.
+
+    Adapted from https://github.com/DIAGNijmegen/picai_prep.
+
+    Args:
+        image (sitk.Image): Image to be resized.
+        spacing (Sequence[float]): Spacing between voxels along each dimentsion.
```

Comment:
It looks like this spacing has an ordering as well, but we don't note it here, as we do with size below.